mdm/orientdb-wrapper/src/main/java/com/gentics/mesh/core/data/generic/MeshVertexImpl.java

```diff
@@ -256,29 +260,47 @@ protected <S extends SortingParameters> S mapSorting(S sorting) {
 		sorting.clearSort();
 		items.entrySet().stream().forEach(entry -> {
 			String key = mapGraphQlFieldNameForSorting(entry.getKey());
-			String[] keyParts = key.split("\\.");
-			if (keyParts.length > 1) {
-				// We expect format of 'fields.<schema_name>'.<field_name>
-				if ("fields".equals(keyParts[0])) {
-					if (keyParts.length > 2) {
-						HibSchema schema = Tx.get().schemaDao().findByName(keyParts[1]);
-						if (schema == null) {
-							throw new IllegalArgumentException("No schema found for sorting request: " + key);
+			String[] fieldParts = key.split("fields\\.");
```

Comment:
We are reassembling the sorting request from the REST/GraphQL representation into the database field structure.